src/xdocs/cmdline.xml.vm

```diff
@@ -203,7 +201,7 @@ java -jar target/checkstyle-X.X-SNAPSHOT-all.jar -c /sun_checks.xml MyClass.java
           <code>docs/sun_checks.xml</code> on a filesystem
         </b>
       </p>
-      <source>
+      <source class="wrap-content">
```

Comment:
if you put wrap-content, then in content of tag should not be any " \". Fix all cases